app/Components/Card.js

```diff
@@ -0,0 +1,17 @@
+import Link from 'next/link';
+
+const Card = ({ iconSrc, title, link }) => {
+  return (
+    <Link href={link}>
+      <div className="bg-green-100 rounded-lg p-4 flex flex-col items-center justify-between h-48 w-48">
```

Comment:
the height and width of the cards should be smaller as there is space between them and the end of the screen refer to figma